src/app/modules/verification/verification.controller.ts

```diff
@@ -283,23 +288,27 @@ export const _handleGenerateOtp: ControllerHandler<
         }
       })
       .andThen((form) =>
-        generateOtpWithHash(logMeta, SALT_ROUNDS)
-          .andThen(({ otp, hashedOtp }) =>
-            // Step 3: Send otp
+        generateOtpWithHash(logMeta, SALT_ROUNDS).andThen(
+          ({ otp, hashedOtp, otpPrefix }) =>
+            // Step 3: Send Otp
```

Comment:
side note: why did you capitalise the O :joy:

Reply:
Ah you discovered my secret ploy to get more LOC changes 💀   /s. 

Not sure why I did that 😆

Reply:
Wah okay I should start changing up all the comments slowly in my PRs too 👀 (jkjk)